src/Core/Content/Product/SalesChannel/Detail/ProductDetailRoute.php

```diff
@@ -172,7 +172,7 @@ private function findBestVariant(string $productId, SalesChannelContext $context
         $criteria = (new Criteria())
             ->addFilter(new EqualsFilter('product.parentId', $productId))
             ->addSorting(new FieldSorting('product.price'))
-            ->addSorting(new FieldSorting('product.available'))
+            ->addSorting(new FieldSorting('product.available', 'DESC'))
```

Comment:
In FieldSorting there should be a constant for DESC

Reply:
changed